invoice/models.py

```diff
@@ -153,14 +153,15 @@ class InvoiceItem(Model):
     """Line item of an invoice."""
     name = CharField(max_length=120)
     description = CharField(max_length=1000)
-    quantity = IntegerField(
-        validators=[MinValueValidator(0), MaxValueValidator(MAX_VALUE_DJANGO_SAVE)])
+    quantity = DecimalField(max_digits=19, decimal_places=2,
```

Comment:
I think we can allow for more decimal places. I'm thinking of 0.001 kg.
So probably something like 5 should do for most use case.